package.json

```diff
@@ -23,6 +23,7 @@
     "@babel/preset-react": "^7.12.1",
     "@types/xml": "^1.0.3",
     "app-root-path": "^2.1.0",
+    "axios": "^0.26.1",
```

Comment:
You installed axios package but you never use it. I recommend you to learn Axios. very nice library for fetch